src/style.scss

```diff
@@ -219,3 +219,18 @@ textarea {
     color: var(--header-bg);
   }
 }
+
+.block-container {
```

Comment:
We don't need to create new style for this, it's better to use primer-css classes.

Reply:
yes well, this was `style="height: 250px; margin-bottom: 24px !important;"` before and so we need to switch it to non inline css, also not everything we can have it in primecss